tools/benchmark/parameters.py

```diff
@@ -1,7 +1,7 @@
 import sys,argparse
```

Comment:
Any plans to publish `benchmark_app.py` ? Cannot find it

Reply:
@dkurt it is here https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tools/benchmark_tool/benchmark_app.py